src/main/resources/static/login.js

```diff
@@ -0,0 +1,13 @@
+    var $input = $("input"),
```

Comment:
We should use a more meaningful variable name here. Rather than ```$input```, consider perhaps ```inputs```?